index.html

```diff
@@ -657,10 +657,10 @@
 [=user agent=], which is to say that it is equivalent to a robot that would carry out a
 [=person=]'s bidding by pressing an [=opt-out=] button with every interaction that the
 [=person=] has with a site, or more generally conveys an expression of the [=person=]'s
-rights in a relevant jurisdiction. (For instance, under the [[?GDPR]], the [=person=] may be
-conveying objections to [=processing=] based on legitimate interest or the withdrawal of
-[=consent=] to specific [=purposes=].) It should be noted that, since a [=global opt-out=]
-signal is reaffirmed automatically with every interaction, it will take precedence
+rights in a relevant jurisdiction. (For instance, under data protection regimes based on legitimate
+interest or consent, the [=person=] may be conveying objections to [=processing=] based on legitimate
+interest or the withdrawal of [=consent=] to specific [=purposes=].) It should be noted that, since a
```

Comment:
If we list things that a global opt-out can do, we should include the thing Sec-GPC [actually does](https://privacycg.github.io/gpc-spec/#the-sec-gpc-header-field-for-http-requests), which is to convey a preference that the person's data not be sold to or shared.
@dmarti's suggestion in https://github.com/w3ctag/privacy-principles/issues/232#issuecomment-1480016252 would also be fine.

Reply:
I like this better than my suggestion -- GDPR, as far as I know, does not mandate a specific format or protocol for an objection to processing, so IMHO it is fine for us to recommend treating GPC as an objection in general.

(This version helps make this document more future-proof -- we don't want it to be too tied to this weird time during which a lot of GDPR "compliance" is still fake, and the cases to close the loopholes haven't worked their way through the process yet)

Reply:
```suggestion
rights in a relevant jurisdiction. (For instance, the [=person=] may be objecting to [=processing=]
based on legitimate interest, withdrawing [=consent=] to specific [=purposes=], or requesting that
their data not be sold or shared.) It should be noted that, since a
```